freqtrade/freqai/data_drawer.py

```diff
@@ -110,7 +139,18 @@ def load_drawer_from_disk(self):
                 "sending null values back to strategy"
             )
 
-        return exists
+    def load_metric_tracker_from_disk(self):
+        """
+        Tries to load an existing metrics dictionary if the user
+        wants to collect metrics.
+        """
+        if self.freqai_info.get('write_metrics_to_disk', False):
+            exists = self.metric_tracker_path.is_file()
+            if exists:
+                with open(self.metric_tracker_path, "r") as fp:
+                    self.metric_tracker = json.load(fp)
```

Comment:
If we write with rapidjson - we should also use rapidjson (with the same flags) for loading to avoid problems and data-discrepancies after a few rounds of read/write (multiple restarts, basically).

Reply:
Done!